### PR TITLE
Automated cherry pick of #1875: Require VACs to use SI units

### PR DESCRIPTION
**Pull request by travisyx**
Cherry pick of #1875 on release-1.15.

#1875: Require VACs to use SI units

For details on the cherry pick process, see the [cherry pick requests](https://git.k8s.io/community/contributors/devel/sig-release/cherry-picks.md) page.

```release-note
This feature changes VolumeAttributesClass, so throughput specifications will need "Mi".
```

### CODE REVIEW

File: test/e2e/tests/single_zone_e2e_test.go
Comment (amacaskill) [inline test/e2e/tests/single_zone_e2e_test.go:1]: This has a lot more changes than what your original PR had .I also don't think you need to change this at all. Please either revert this, or just only change the 2 lines that were in the original file

File: examples/kubernetes/demo-vol-create.yaml
Comment (amacaskill) [inline examples/kubernetes/demo-vol-create.yaml:1]: I don't think this is needed either.

Comment (travisyx) [reply examples/kubernetes/demo-vol-create.yaml:1]: Done

File: test/e2e/tests/single_zone_e2e_test.go
Comment (travisyx) [reply test/e2e/tests/single_zone_e2e_test.go:1]: I reverted this, the original SI change modified tests added in a PR that is not included in v1.15.

**Pull request merged**